rascal-vscode-extension/src/auto-jvm/JavaLookup.ts

```diff
@@ -102,7 +102,7 @@ function getJavaCandidates(): string[] {
             }
         }
     }
-    return result;
+    return [];//result;
```

Comment:
is this right?

Reply:
nope, this is debugging code, I wanted to trigger it on my machine. good find.